### PR TITLE
Minor doc updates

### PR DESCRIPTION
**Pull request by suneet-s**
Update kafka ingestion specs in tutorial docs to use the new inputSpec instead of parseSpec
Update first/ last aggregator docs to remove filterNullValues

### CODE REVIEW

File: docs/querying/aggregations.md [hunk 1]
```diff
@@ -231,8 +230,7 @@ Note that queries with first/last aggregators on a segment created with rollup e
   "type" : "stringLast",
   "name" : <output_name>,
   "fieldName" : <metric_name>,
-  "maxStringBytes" : <integer> # (optional, defaults to 1024),
-  "filterNullValues" : <boolean> # (optional, defaults to false)
+  "maxStringBytes" : <integer> # (optional, defaults to 1024)
 }
```
Comment (suneet-s) [inline docs/querying/aggregations.md]: FYI @maytasm3

File: examples/quickstart/tutorial/wikipedia-kafka-supervisor.json [hunk 2]
```diff
@@ -3,46 +3,46 @@
...
+            "name": "deleted",
+            "type": "long"
+          },
+          {
+            "name": "delta",
+            "type": "long"
+          }
+        ]
+      },
       "parser": {
```
Comment (jihoonson) [inline examples/quickstart/tutorial/wikipedia-kafka-supervisor.json]: You can remove the `parser` here.

File: docs/querying/aggregations.md (hunk 1 above)
Comment (maytasm3) [reply docs/querying/aggregations.md]: Thanks. Looks good

File: examples/quickstart/tutorial/wikipedia-kafka-supervisor.json (hunk 2 above)
Comment (suneet-s) [reply examples/quickstart/tutorial/wikipedia-kafka-supervisor.json]: 🤦‍♂ done

**Pull request merged**